xml/pop_banners.xml

```diff
@@ -225,6 +225,20 @@ matched against these patterns to fingerprint POP3 servers.
       <param pos="0" name="os.certainty" value="0.5"/>
    </fingerprint>
 
+   <fingerprint pattern="^TCPIP POP server V\d\.\d\S-\S{3}, OpenVMS V(\d\.\d-\d) .*$">
+      <description>TCP/IP Services for OpenVMS POP server</description>
+      <example os.version="7.3-2">TCPIP POP server V5.4J-15A, OpenVMS V7.3-2 Alpha at example.com, up since 2015-02-12 08:44:53 20400434.2</example>
```

Comment:
Everything after the version -- is that consistently there?  If so, you should extract the hostname as `host.name` or domain as `host.domain` depending on what example.com is actually representing.